### DISCUSSION TITLE
Speedhack lag

### DISCUSSION THREAD

**Discussion by Davie787** [maxnut/GDmenu#278] [Q&A]
When I activate speedhack and slow it down to like 0.1 my fps suddenly lowers and it stays that way until I disable my speedhack again. But it only happens when I am playing a level, in the main menu the icons slide by just fine. I tried looking it up but I couldn't find anyone else with the same problem. Can someone please help me?

**Comment by GamingFloppaYT** [comment]
i have the same thing too, i think its just when you lower the speed to very low the frames become more noticeable

**Reply by Davie787** [reply]
Yeah, but when 2.1 was still here this wasn't a thing so I am thinking I maybe changed a setting or something because it didn't use to do that. Also I don't understand why the main menu is smooth and playing a level isn't.
